### ISSUE TITLE
Group in Alarmo (2x Doors) trigger immediately instead of after consecutive opens.

### ISSUE THREAD

**Issue by vaemarr** [nielsfaber/alarmo#933]
### Alarmo Version

1.10.0

### HA Version

2024.4.3

### Bug description

I have an alarmo group consisting of 2x door sensors - my garage door and another inner door in the garage.

Both have 'Use entry delay enabled'.

Alarmo Group timeout is set to 5 mins. Both sensors are selected for the group.

I only need to activate one sensor (open the door) before the alarm is triggered. Instead of having to activate another sensor within the minute window as you would expect.

I can't workout why this is happening.

### Steps to reproduce

Setup 2 testing scenarios. One to only open the inner door - the other to open the garage.

Waited for 15 mins between each test to be sure no timeout was active.

Checked group settings to be sure setup is correct.

Opened different doors in each test, but only one door.

Alarmo is immediately triggered despite a 2nd activation not occuring.

### Relevant log output

```shell
Not sure where to get this.
```

**Comment by nielsfaber** [comment]
In an attempt to reproduce this issue, I created a sensor group like this:
<img src="https://github.com/nielsfaber/alarmo/assets/16861605/1b7b35df-506c-4e4d-b152-e55211a36dbd" width="75%" height="75%">

Both sensors are configured to be active in the 'armed home' and 'armed away'.
With all sensors closed, I arm the alarm in mode 'armed away'. When the system is armed I open the front door.
The debug log now shows:
```
2024-05-08 14:14:25.575 DEBUG (MainThread) [custom_components.alarmo.sensors] entity binary_sensor.frontdoor changed: old_state=closed, new_state=open
2024-05-08 14:14:25.575 DEBUG (MainThread) [custom_components.alarmo.sensors] tripped sensor binary_sensor.frontdoor was ignored since it belongs to group My Group
```
The alarm remains in state 'armed away'.
Next I open the front door.
The debug log now shows:
```
2024-05-08 14:18:06.409 DEBUG (MainThread) [custom_components.alarmo.sensors] entity binary_sensor.back_door changed: old_state=unavailable, new_state=open
2024-05-08 14:18:06.410 DEBUG (MainThread) [custom_components.alarmo.sensors] tripped sensor binary_sensor.back_door caused the triggering of group My Group
2024-05-08 14:18:06.410 INFO (MainThread) [custom_components.alarmo.sensors] Alarm is triggered due to sensor: binary_sensor.back_door
2024-05-08 14:18:06.410 DEBUG (MainThread) [custom_components.alarmo.alarm_control_panel] entity alarm_control_panel.alarmo was updated from armed_away to pending
2024-05-08 14:18:06.411 INFO (MainThread) [custom_components.alarmo.alarm_control_panel] Alarm will be triggered after 5 seconds.
```

In summary, opening the first door is ignored, the second one triggers the alarm. 
This works as the function is intended to work.

I cannot reproduce the problem you are experiencing. 
Please provide more information if you think something is wrong within Alarmo.

If you wish to see debug logging, you can do so by adding this to the configuration.yaml:
```
logger:
  default: warning
  logs:
    custom_components.alarmo: debug
```
The extra information should appear in the HA logs.

**Comment by sato9882** [comment]
Alarmo Version
1.10.1

HA Version
2024.5.2

Bug description
Same behavior as vaemarr.
Opening just one door will trigger an alarm.
